Vagrantfile

```diff
@@ -345,8 +346,9 @@ Vagrant.configure(2) do |config|  # the literal "2" is required.
  # . . . . . . . . . . . . Define machine win16 . . . . . . . . . . . . . .
  # . this machine installs Salt on a Windows 2016 Server and runs highstate.
   config.vm.define "win16", autostart: false do |quail_config|
-    quail_config.vm.box = "gusztavvargadr/w16s" # Windows Server 2016 standard
-    quail_config.vm.box_version = "1808.0.0" # WARNING: 1809.0.0 does not have WinRM enabled
+    quail_config.vm.box = "cdaf/WindowsServer" #gusztavvargadr/w16s" # Windows Server 2016 standard
+    # "jacqinthebox/windowsserver2016"
+    # quail_config.vm.box_version = "1808.0.0" # WARNING: 1809.0.0 does not have WinRM enabled
```

Comment:
Should these comments be deleted too?

Reply:
I was lazy. These are other VM package that I have tried and though I might want to revert to.
I'll pull the comments now and let it come through on the next revision.